src/kedro_templar/process_templates.py

```diff
@@ -0,0 +1,145 @@
+import logging
+import os
+from pathlib import Path
+from typing import Text, Dict
+import boto3
+from botocore.exceptions import ClientError
+import click
+import yaml
+from jinja2 import Environment, FileSystemLoader
+
+log = logging.getLogger(__name__)
+
+
+# settings.py
+TEMPLATES_DIR = Path('./examples/base')
+OUTPUT_DIR = Path("./conf/base")
+aws_access_key = os.environ.get('AWS_ACCESS_KEY_ID')
+aws_secret_key = os.environ.get('AWS_SECRET_ACCESS_KEY')
+aws_region = os.environ.get('AWS_DEFAULT_REGION')
+
+# lib/templates.p
+def create_environment(templates_dir: Text) -> Environment:
```

Comment:
Move to separate file `src/kedro_templar/lib/templates.py`

Reply:
ok

Reply:
ok